streaming/java/streaming-api/src/main/java/io/ray/streaming/operator/SourceOperator.java

```diff
@@ -4,7 +4,7 @@
 
 public interface SourceOperator<T> extends Operator {
 
-  void run();
```

Comment:
'run' seems a better name than 'fetch' for me.

Reply:
Run seems this method will never stop. I think fetch is better.